Week3/HW3/3_moreJavascript/13_moduleTeachers.js

```diff
@@ -0,0 +1,8 @@
+let moduleTeachers = {};
+moduleTeachers = [
+    { name: 'Philipp Beau' },
+    { name: 'Rob van Kruijsdijk' },
+    { name: 'Unmesh Joshi' },
+    { name: 'Yash Kapila' },
+    {name: 'Wilgert Velinga'}
+  ];
```

Comment:
Ah. This isn't what was expected out of the exercise. If you have a look at the homework again, you were supposed to create an object of teachers and the modules which they have taught so far. Even though JavaScript treats an `array` as an `object`, using an array here wouldn't be so useful. Also, if you have a look again at your code, you created `moduleTeachers` as an object but later on changed it to an array. I think a better way of doing it would be like:

```
const moduleTeachers = {
  'HTML': 'Philipp Beau',
  'CSS': 'Rob van Kruijsdijk',
  'CLI': 'Unmesh Joshi',
  // and so on
};
```

What do you think?

Reply:
Yea. I corrected them  as well.
i didn't even notice. 
Pardon me also, please,  because i am very new to this area and trying to do my best to stay in, despite of discouragements around me. 😞

I do appreciate your efforts.